index.js

```diff
@@ -12,8 +15,32 @@ formatCustomerBalance("Jane Doe", 250); // => "John Doe has a balance of 250 KD.
 
 function getBooksPublishedAfter2010(books) {
   // Write code here
+  // const book = getBooksPublishedAfter2010.fitler()
+  // books[1].year >= 2012;
+  // return books[1];
+  // return (books) => books.year >= 2010;
+  // return (books = books.year >= 2010);
+  books.forEach((books) => {
+    console.log(books.year);
```

Comment:
you are only logging the value of years, you should return an array containing the book titles which are published after 2010